postman/Manage Custom Fields.postman_collection.json

```diff
@@ -0,0 +1,407 @@
+					"host": [
+						"api",
+						"aweber",
+						"com"
+					],
+					"path": [
+						"1.0",
+						"accounts"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Lists",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "f6163948-14a0-48e5-ad18-ee37ab1a3c0d",
+						"exec": [
+							"pm.test(\"Successfully get a  random list\", function() {",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"pm.test(\"At least one list returned\", function() {",
+							"    var jsonData = pm.response.json();",
+							"    pm.expect(jsonData.total_size).to.be.above(0);",
+							"    pm.globals.set(\"listId\",jsonData.entries[0].id );",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "https://api.aweber.com/1.0/accounts/{{accountId}}/lists",
+					"protocol": "https",
+					"host": [
+						"api",
+						"aweber",
+						"com"
+					],
+					"path": [
+						"1.0",
+						"accounts",
+						"{{accountId}}",
+						"lists"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get the First List",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "75df1ad8-7a66-40eb-b8e7-2991fcd51619",
+						"exec": [
+							"pm.test(\"Successfully get a list\", function(){",
+							"    pm.response.to.have.status(200);",
+							"});",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_URL}}/accounts/{{accountId}}/lists/{{listId}}",
+					"host": [
+						"{{base_URL}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Custom Fields",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "b5a97102-9d21-4722-a899-3444de45eb5b",
+						"exec": [
+							"var jsonData =  pm.response.json();",
+							"",
+							"pm.test(\"Get custom fields in list\", function(){",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"pm.test(\"One or more custom fields returned\", function() {",
+							"    pm.expect(jsonData.total_size).to.be.above(0);",
+							"});",
+							"",
+							"pm.test(\"New custom field is not empty and not null\", function() {",
+							"   pm.expect(pm.variables.get(\"custom_field\")).not.eql(\"\") && pm.expect(pm.variables.get(\"custom_field\")).not.eql(null);",
+							"});",
+							"",
+							"pm.test(\"New custom field is unique\", function() {",
+							"    for ( i = 0; i < jsonData.entries.length; i++ ){",
+							"        pm.expect(jsonData.entries[i].name).to.not.include(pm.variables.get(\"custom_field\") );",
```

Comment:
This is too broad. If I have an existing custom field called "newfield" and custom_field = "field", this test fails.